doc/source/release-notes/Percona-Server-5.6.35-80.1.rst

```diff
@@ -0,0 +1,63 @@
+.. rn:: 5.6.35-80.1
+
+============================
+|Percona Server| 5.6.35-80.1
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.35-80.1 on
+March 24th, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.35-80.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.35
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-35.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.35-80.1 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.35-80.1
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.35-80.1>`_.
+
+Bugs Fixed
+==========
+
+ When :variable:`innodb_ft_result_cache_limit` was exceeded by internal memory
+ allocated by |InnoDB| during the FT scan not all memory was released which
+ could lead to server assertion. Bug fixed :bug:`1634932` (upstream
+ :mysqlbug:`83648`).
+
+ Log tracking initialization did not find last valid bitmap data correctly,
+ potentially resulting in needless redo log retracking or hole in the tracked
+ LSN range. Bug fixed :bug:`1658055`.
+
+ If :ref:`audit_log_plugin` was unable to create file pointed by
+ :variable:`audit_log_file`, server would crash during the startup. Bug fixed
+ :bug:`1666496`.
+
+ ``CREATE TEMPORARY TABLE ... SELECT ...`` would get logged in the binary
+ log on a disconnect with mixed mode replication. Bug fixed :bug:`1671013`.
+
+ |TokuDB| did not use index with partitioned tables even if cardinality was
+ good. Bug fixed :bug:`1671152`.
+
+ Row-based replication events were not reflected in ``Rows_updated`` fields in
+ the :ref:`user_stats` ``INFORMATION_SCHEMA`` tables. Bug fixed :bug:`995624`.
+
+ Server could become unresponsive for some time during the ``Query end`` stage
+ when loading big files using the ``LOAD DATA INFILE``. Bug fixed
+ :bug:`1646100`.
+
+ Compression dictionaries were not being used in partitioned tables. Bug fixed
```

Comment:
It was impossible to use column compression dictionaries with partitioned InnoDB tables